homeassistant/components/ebusd/__init__.py

```diff
@@ -34,6 +35,15 @@ def verify_ebusd_config(config):
     for condition in config[CONF_MONITORED_CONDITIONS]:
         if condition not in SENSOR_TYPES[circuit]:
             raise vol.Invalid("Condition '" + condition + "' not in '" + circuit + "'.")
+    cache_ttl = config[CONF_CACHE_TTL]
+    if cache_ttl < MIN_TIME_BETWEEN_UPDATES.seconds:
+        raise vol.Invalid(
+            "Cache TTL "
+            + str(cache_ttl)
+            + " cannot be less than "
+            + str(MIN_TIME_BETWEEN_UPDATES.seconds)
+            + " seconds."
+        )
```

Comment:
can be removed after that

Reply:
I only have min value (which is a parameter to Throttle), but not sure about max. Documentation for the daemon this integration talks to says:
`-m SECONDS   only return cached value if age is less than SECONDS [300]`